001-cron.php

```diff
@@ -138,5 +138,13 @@ function wpcom_vip_cron_control_event_object_to_string( $event ) {
 	add_action( 'a8c_cron_control_event_threw_catchable_error', 'wpcom_vip_log_cron_control_event_for_caught_error', 10, 2 );
 	add_action( 'a8c_cron_control_freeing_event_locks_after_uncaught_error', 'wpcom_vip_log_cron_control_event_object' );
 
-	require_once __DIR__ . '/cron-control/cron-control.php';
+	$cron_control_next_version = __DIR__ . '/cron-control-next/cron-control.php';
+
+	if ( defined( 'VIP_CRON_CONTROL_USE_NEXT_VERSION' ) && VIP_CRON_CONTROL_USE_NEXT_VERSION && file_exists( $cron_control_next_version ) ) {
```

Comment:
Minor: It's nice to check the explicit value of the constant:

```suggestion
	if ( defined( 'VIP_CRON_CONTROL_USE_NEXT_VERSION' ) && true === VIP_CRON_CONTROL_USE_NEXT_VERSION && file_exists( $cron_control_next_version ) ) {
```

Reply:
Good call, committed.